testutils/assertions.go

```diff
@@ -19,10 +23,9 @@ import (
 //       <string>: "...-10101" ver..."
 func ExpectEqualProtoMessages(a, b proto.Message, optionalDescription ...interface{}) {
 	if proto.Equal(a, b) {
```

Comment:
I think I mislead you with the structure of my pseudocode
function should return with no output if `proto.Equal(a, b)`